Best practices/React/Testing.md

```diff
@@ -98,6 +98,18 @@ Another benefit of testing React components is that they typically abstract away
   describe('<MyComponent />', () => {});
   ```
 
+* When there are multiple tests for the same prop or callback function, they should be wrapped in a single describe block
+
+  ```js
+  describe('<MyComponent />', () => {
```

Comment:
Should we should a "bad" example here?

Reply:
This one is also kind of referenced in https://github.com/Shopify/web-foundation/pull/66/files#diff-eeb8827fd304d15a55965bd445d003e0R9, should we just add the example there?

Reply:
I think it's a good idea to call this one out explicitly with an example.